packages/ckeditor5-find-and-replace/README.md

```diff
@@ -0,0 +1,8 @@
+CKEditor 5 Find and Replace feature
```

Comment:
It seems that we should clarify how all the packages are being named in package.json etc files.  
  
For example - following your example  
[Special-characters](https://github.com/ckeditor/ckeditor5/blob/523f88cfc55a920d931c784ab3469e94f749557e/packages/ckeditor5-special-characters/package.json#L4) in package.json have:  
`"description": "Special characters feature for CKEditor 5.",`  
while, [media-embed](https://github.com/ckeditor/ckeditor5/blob/523f88cfc55a920d931c784ab3469e94f749557e/packages/ckeditor5-media-embed/package.json#L4) or [code-block](https://github.com/ckeditor/ckeditor5/blob/523f88cfc55a920d931c784ab3469e94f749557e/packages/ckeditor5-code-block/package.json#L4) have:  
`"description": "Media Embed feature for CKEditor 5.",`  
`"description": "Code Block feature for CKEditor 5.",`  
  
From what I've seen in LICENSE.md we're using capitalized letters for packages names generally.  
  
In README.md (I wasn't checking all of them of course) I've also noticed that it differs depending on the package:  
[Cloud-services](https://github.com/ckeditor/ckeditor5/blob/523f88cfc55a920d931c784ab3469e94f749557e/packages/ckeditor5-cloud-services/README.md) or [Markdown](https://github.com/ckeditor/ckeditor5/blob/523f88cfc55a920d931c784ab3469e94f749557e/packages/ckeditor5-markdown-gfm/README.md) have capitalized letters while media-embed / code-block does not.  
  
I'd add this as well to the [notion package guide](https://www.notion.so/Creating-a-new-package-f3528d73f2324413a8222ac4e8ec1334#61860e64a0f3473ba93ddfa54c23e3ea).

Reply:
Cloud Services is the name of our bigger product family and we always use capital. Markdown is also a name of "technology" (quotes because it's not a big thing :D), for example see wikipedia [https://en.wikipedia.org/wiki/Markdown](https://en.wikipedia.org/wiki/Markdown) they also refer to it with capitalization.

I checked other plugins and we're surprisingly consistent with the letter casing - its lower case pretty much everywhere else.

Reply:
And yes, it make sense to mention that one should pay attention to feature name casing in our internal documentation.

Reply:
I've created an issue to clean up those differences - #9781